virttest/iscsi.py

```diff
@@ -347,6 +347,9 @@ def cleanup(self):
             restart_iscsid()
         if self.export_flag:
             self.delete_target()
+        cmd = "targetcli clearconfig confirm=true"
+        if process.system(cmd, shell=True) != 0:
+            logging.error("targetcli configuration unable to clear")
```

Comment:
This cleanup operation should not be done by the base class `_IscsiComm`, the right place is somewhere in `IscsiLIO`.

Reply:
Hi luckyh,

i want to clear the configuration before device created, IscsiLIO will effect the after device created.